### DISCUSSION TITLE
2021 拜年纪播放页的布局其实算不算是 B 站官方做的播放器置顶

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1621] [综合讨论 / General]
瞎聊一下。

2021 拜年纪播放页的布局其实是不是可以看成 B 站官方做的播放器置顶。

如果以后有深度优化 `播放器置顶` 功能的计划，感觉可以参考 2021 拜年纪播放页的布局。看着比简单粗暴地把上面的元素拉下来好看得多（现在看着怪怪的可能还是因为 `创作团队` 那个卡片在那个位置稍显突兀而且不对齐）。

见下图。其实整体布局还是差不多的。有那么点参考了 YouTube 的意思。区别有视频标题下方不再是和播放量弹幕量日期那堆基本数据，把操作菜单插在了中间，然后是简介再然后是宽度够拉伸后看着舒服些的创作团队。下面的横幅我觉得可以对应 tag 栏或者活动横幅（如果不管最上面的拜年纪横幅的话。上面这位置放日常视频的 tag 或活动横幅都感觉怪怪的）
![image](https://user-images.githubusercontent.com/49396421/110152047-31864800-7e1c-11eb-9b59-d6979adc2acb.png)

**Comment by the1812** [comment]
播放页那里左右两个板块是分开的, 我只能各自上下移动他们, 像是把 up 拉到左边就不行了

**Reply by Geeyun-JY3** [reply]
就瞎聊一下，感觉挺有意思的，官方做了这么个布局出来
